contracts/modules/SSVOperators.sol

```diff
@@ -129,6 +135,8 @@ contract SSVOperators is ISSVOperators {
             revert ApprovalNotWithinTimeframe();
         }
 
+        if (feeChangeRequest.fee.expand() > SSVStorageProtocol.load().operatorMaxFee) revert FeeTooHigh();
```

Comment:
@andrew-blox when executing the fee change, here I check again if the fee exceeds the limit. It can be changed between declaring and executing the change.

Reply:
Got it!  Good catch